users/README.md

```diff
@@ -101,6 +107,12 @@
     "restricted": false,
     "super_user": false,
     "in_discord" : true,
+  },
+    "archivedDetails": {
+    "archived_at": "String",
+    "archived_user_id": "String",
+    "super_user_id": "String",
+    "reason": "String"
```

Comment:
this is a example you should give the values no the data type!

Reply:
updated, my bad